pkg/pf/tfbridge/provider_delete.go

```diff
@@ -52,13 +53,17 @@ func (p *provider) DeleteWithContext(
 		return resource.StatusOK, err
 	}
 
+	plannedState, err := tfprotov6.NewDynamicValue(tfType, tftypes.NewValue(tfType, nil))
+	contract.AssertNoErrorf(err, "nil is always a valid value to marshal to a dynamic state")
+
 	// terraform-plugin-framework recognizes PlannedState=nil ApplyResourceChangeRequest request as DELETE.
 	//
 	//nolint:lll // See
 	// https://github.com/hashicorp/terraform-plugin-framework/blob/ce2519cf40d45d28eebd81776019e68d1bddca6f/internal/fwserver/server_applyresourcechange.go#L63
 	req := tfprotov6.ApplyResourceChangeRequest{
-		TypeName:   rh.terraformResourceName,
-		PriorState: priorState,
+		TypeName:     rh.terraformResourceName,
+		PriorState:   priorState,
+		PlannedState: &plannedState,
```

Comment:
How does this affect PF? I'd love to see some tests there to prove it doesn't.

Reply:
I'm pretty sure deletion lifecycle is covered for PF, but let me poke in the cov reports.